src/ucp/wireup/wireup_cm.c

```diff
@@ -271,8 +272,11 @@ static unsigned ucp_cm_client_connect_progress(void *arg)
 
     ucs_assert(addr.address_count <= UCP_MAX_RESOURCES);
     ucs_assert(wireup_ep->ep_init_flags & UCP_EP_INIT_CM_WIREUP_CLIENT);
-    status = ucp_wireup_init_lanes(ucp_ep, wireup_ep->ep_init_flags, &addr,
-                                   addr_indices);
+    tl_bitmap = ucp_ep_get_tl_bitmap(ucp_ep);
+    /* EP lanes must be configured to right device on previous stage */
+    ucs_assert(tl_bitmap & worker->context->tl_bitmap);
```

Comment:
maybe assert inside ucp_wireup_init_lanes that the tl_bitmap is not 0

Reply:
in addition or instead of this one?